packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -126,13 +131,15 @@ const useLiveChat = (has_cookie_account = false, active_loginid?: string) => {
     }, [history, isMounted, onHistoryChange]);
 
     useEffect(() => {
-        if (reload) {
+        if (reload && !should_not_run_livechat) {
             liveChatSetup(has_cookie_account);
             setReload(false);
         }
     }, [reload, has_cookie_account]);
 
-    useEffect(() => liveChatSetup(has_cookie_account), [has_cookie_account, active_loginid]);
+    useEffect(() => {
+        if (!should_not_run_livechat) liveChatSetup(has_cookie_account);
+    }, [has_cookie_account, active_loginid]);
```

Comment:
shouldnt we `should_not_run_livechat` to the dependencies

Reply:
will add it there 🙏